recipes/epics-base/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "3.14.12.8" %}
+{% set sha256 = "b11eab2fdd01c60219be48954631f22230107380961e0df7c356f7cc8bb5f24e" %}
+
+package:
+  name: epics-base
+  version: {{ version }}
+
+source:
+  fn: baseR{{version}}.tar.gz
+  url: https://epics.anl.gov/download/base/baseR{{version}}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - config_site.diff  # [not win]
+    - config_common.diff  # [not win]
+    - rules_build.diff
+    - config_readline.diff  # [linux]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make  # [not win]
+    - m2w64-make  # [win]
+    - perl  # [win]
+
+test:
+  commands:
+    - test -f ${PREFIX}/etc/conda/activate.d/epics_base.sh  # [not win]
+    - test -f ${PREFIX}/etc/conda/deactivate.d/epics_base.sh  # [not win]
+    - test -f ${PREFIX}/epics/include/epicsVersion.h  # [not win]
+
+about:
+  home: http://www.aps.anl.gov/epics
+  license: Epics Open
+  license_file: LICENSE
+  summary: EPICS Base Library
+
+extra:
+  recipe-maintainers:
+    - simongregorebner
+    - xiaoqiangwang
```

Comment:
@simongregorebner, @xiaoqiangwang just to confirm, you are okay with being package maintainers?

Reply:
thats ok with me